packages/oui-stepper/src/step-form/step-form.controller.js

```diff
@@ -0,0 +1,59 @@
+import { addBooleanParameter, addDefaultParameter } from "@oui-angular/common/component-utils";
+
+export default class StepFormController {
+    constructor ($attrs, $element, $scope, $timeout, ouiStepperConfiguration) {
+        "ngInject";
+
+        this.$attrs = $attrs;
+        this.$element = $element;
+        this.$scope = $scope;
+        this.$timeout = $timeout;
+        this.translations = angular.copy(ouiStepperConfiguration.translations);
```

Comment:
why a copy? it's should never be edited outside provider

Reply:
IMHO we can assume that our components never modify it and don't copy it.. 
If it's required for this component, it should be required for all others too. I prefer it in the provider.
I let you choose :)